src/views/tinode-web.jsx

```diff
@@ -1428,6 +1445,32 @@ class TinodeWeb extends React.Component {
     });
   }
 
+  handleShowForwardMenu(params) {
+    if (this.state.sidePanelSelected == 'newtpk') {
+      // If the Find panel is active, close it.
+      this.handleSidepanelCancel();
+    }
+    const header = "Forwarded from " + params.userName;
```

Comment:
Internationalize?

Reply:
Strip previous Forwarded for header?
<img width="226" alt="Screenshot 2021-09-21 at 11 16 51" src="https://user-images.githubusercontent.com/5938143/134225959-2d41d79c-4cd2-40e3-9b73-8af3709a10a2.png">